lib/ghcjs-project.nix

```diff
@@ -38,8 +53,19 @@ let
             alex
             cabal-install
         ];
+    # nixpkgs does not have an emsdk, this derivation uses symlinks to make something
```

Comment:
That's curious. I though there was something, but it might also have been outdated :-(